### ISSUE TITLE
Adjust ALS-A degree requirement to check for 1.0 credits instead of 1 course

### ISSUE THREAD

**Issue by hawkrives** [hawkrives/gobbldygook#2333]
… even though the catalog says "1 course", apparently the rule is "some number of courses that add up to 1 credit".

**Comment by rye** [comment]
Yes, that's correct. Music lessons are 0.25 credits each, and you need four semesters to call your ALS-A fulfilled.

**Comment by hawkrives** [comment]
Yeah, but the catalog doesn't agree. I'm just grumpy.
